pkg/helm/install_chart_test.go

```diff
@@ -0,0 +1,32 @@
+package helm
+
+import (
+	"github.com/stretchr/testify/assert"
+	"helm.sh/helm/v3/pkg/action"
+	"helm.sh/helm/v3/pkg/chartutil"
+	kubefake "helm.sh/helm/v3/pkg/kube/fake"
+	"helm.sh/helm/v3/pkg/storage"
+	"helm.sh/helm/v3/pkg/storage/driver"
+	"io/ioutil"
+	"testing"
+)
+
+func TestInstallChart(t *testing.T) {
+	chart := "https://srv-file5.gofile.io/download/dtv19k/influxdb-3.1.1.tgz"
```

Comment:
we should avoid depending on external resources, if possible, because there could a network issue during testing or resources could simply disappear. 

I would try to make a real unit test here - instead of integration tests we have now. Just to test if our logic is ok, i.e. if helm functions are invoked properly.